docs/samples/rollouts/promotion.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: "serving.kubeflow.org/v1alpha2"
+kind: "InferenceService"
+metadata:
+  name: "my-model"
+spec:
+  # This is now the promoted canary model
+  default: 
+    predictor:
+      tensorflow:
+        storageUri: "gs://kfserving-samples/models/tensorflow/flowers-2"
+  # Defaults to zero, so can also be omitted or explicitly set to zero.
+  canaryTrafficPercent: 0
+  # This use to be the default model
+  canary:
+    predictor:
+      # Canary is created but no traffic is directly forwarded.
+      tensorflow:
+        storageUri: "gs://kfserving-samples/models/tensorflow/flowers"
```

Comment:
it is a bit odd that canary uses the old model now, we can just remove the canary section so that canary pods get deleted after promotion.